CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Unreleased
+
+## Fixes
+
+- [Pull request #1118: Fix code sample accessibility issues](https://github.com/alphagov/govuk-prototype-kit/pull/1118)
```

Comment:
we don't normally update the changelog for the documentation as it is released immediately on merge, not as part of the release

Reply:
Fixed.